preprocess.py

```diff
@@ -52,16 +57,29 @@ def preproc_str(string: str, replaceContractions: bool = False) -> str:
     """
     string = string.lower()
     string = __remove_new_line(string)
-    if(replaceContractions):
+    if (replaceContractions):
         string = __remove_contractions(string)
     string = __remove_punctuation(string)
     string = __remove_whitespaces(string)
     return string
 
+
 def get_word_count(string: str) -> int:
     """
     Returns an integer indicating the number of words in the string.
     Uses preproc_str() first to ensure special characters are not counted as words.
     """
     arr_str = preproc_str(string).split(" ")
-    return len(arr_str)
\ No newline at end of file
+    return len(arr_str)
+
+
+def sort_text(text: str) -> list:
+    """
+    This function sorts the inputted text string
+    :param text: A preprocessed string
```

Comment:
for our project, we'll need to make sure we're consistent with function comments about parameters and returns! This looks good

Reply:
Yes! With typescript we will need to be consistent as well, so it's important to note this